pages/api/EventsService.ts

```diff
@@ -0,0 +1,56 @@
+/*
+* @2023 Open Seattle
+*/
+
+import EventPlaceholder from '../../assets/event-placeholder.png';
+import localEvents from '../../assets/events.json';
+
+class OsEventTime {
+    start: string;
+    end: string;
+
+    constructor(data: any) {
+        this.start = data.start;
+        this.end = data.end;
+    }
+}
+
+class OSEvent {
+    title: string;
+    date: string;
+    time: OsEventTime;
+    location: string;
+    description: string;
+    imageSrc: string;
+    imageAlt: string;
+    buttonLink: string;
+    active = true;
+
+    constructor(data: any) {
+        this.title = data.title;
+        this.date = data.date;
+        this.time = new OsEventTime({ start: data.start, end: data.end });
+        this.location = data.location;
+        this.description = data.description ? data.description : '';
+        this.imageSrc = data.imageSrc ? data.imageSrc : EventPlaceholder.src;
+        this.imageAlt = data.imageAlt;
+        this.buttonLink = data.buttonLink;
+    }
+}
+
+class OSEventService {
```

Comment:
nice! though we'll likely be using Sanity as a way to let anyone publish an event, or add any kind of content, like cadres members, volunteer members, and project information.  I'm fine checking in this for now and modify it later when we incorporate sanity.

Reply:
The OSEventService can serve as a facade to Sanity.  That would allow mocking in a dev environment (I am guessing connecting to Sanity can be challenging).  Also, mocking the events would make it easier to develop for corner cases (e.g. events without images or descriptions that are too long or have invalid characters).